### ISSUE TITLE
OGC standards could allow for different levels of detail/complexity to accommodate basic users who may only be interested in a subset of the capabilities offered by a standard

### ISSUE THREAD

**Issue by ogcscotts** [ogcscotts/TC-Meeting-topics#92]

**Comment by pebau** [comment]
yes, they do already - this is exactly why the Modular Specification model has been established. Works like a charm!

**Comment by jyutzler** [comment]
No, it does not. The Modular Specification was designed for rules and regulations people, not developers. In GeoPackage we found that the only reasonable way to handle this was by having an independent set of informative documents that parallels the standard. On geopackage.org, we currently have the following:
* http://www.geopackage.org/spec121/ - the latest version of the adopted standard
* http://www.geopackage.org/spec/ - the working version, which includes things that have been agreed on in principle, but not yet adopted
* http://www.geopackage.org/guidance/getting-started.html - the getting started guide, designed for new developers and those who will want to interact with GeoPackages directly
* http://www.geopackage.org/guidance/implementation_guide.html - the implementation guide, designed for developers who desire to implement specific parts of the standard

**Comment by pebau** [comment]
Might be that not for all standards it does the expected thing, I humbly emphasize that this I cannot oversee. I just can report from the experience with WCS where it works actually, underlined by the pretty good acceptance in the communities:
- Core: "give me that coverage, or a subset thereof, in my preferred format".
- Extensions at your choice for all sort of functionality required, starting from simple range subsetting ("give me bands A, B, and C") over CRS transformation up to analytics.
- bespoke Application Profiles, such as for remote sensing and MetOcean.

So basic users can always just look at the simplest thing you may want to do with a coverage at all.

**Comment by chris-little** [comment]
I think that the push towards Open APIs and service architectures should enable more effective hiding of complexity from users only interested in an initial simple request. Consequently, we should not fall into the trap on having a 'single' OpenAPI service replacing the current WxS services, but a hierarchy or network of services to meet varied requirements.
